### PR TITLE
Feature/mark dr accepted

### PR DESCRIPTION
**Pull request by dp6ai**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -46,5 +46,6 @@ class Application < Rails::Application
     # Feedback URL (URL for feedback link in phase banner)
     # Use 'auto_add_path' for it to add a path link to the new_feedback route
     config.feedback_url = config.relative_url_root + '/feedback/new'
+    config.action_mailer.default_url_options = { host: 'example.com' } #TODO: this needs correcting
```
Comment (ansonK) [inline config/application.rb]: shouldn't this be an ENV variable?

Comment (dp6ai) [inline config/application.rb]: It can be. I wanted to add a TODO and the settings.yml would have been unhappy. It can be fixed later i think

**Comment by FriedSock** [comment]
Quite a lot of logic in the controller, we could ideally add some fancy validations to the model instead but I don't think it is worth the time to refactor at this point if we are "rebuilding" in a few days.

**Comment by dp6ai** [comment]
@FriedSock completely agree, the refactor can wait until after alpha

**Pull request merged**